src/operations/search/query/r4.js

```diff
@@ -122,7 +122,23 @@ module.exports.buildR4SearchQuery = (resource_name, args) => {
         columns.add('meta.lastUpdated');
     }
 
-    for (const [field, filterObj] of Object.entries(customQueries)) {
+    for (const [resourceType, filterObj] of Object.entries(customScalarQueries)) {
+        if (resourceType === resource_name) {
```

Comment:
Minor thing with the variable naming conventions of camel vs underscore.

Reply:
fixed